tools/trace_analyzer_tool.cc

```diff
@@ -169,6 +168,8 @@ DEFINE_double(sample_ratio, 1.0,
 
 namespace ROCKSDB_NAMESPACE {
 
+const size_t kShadowValueSize = 10;
```

Comment:
Maybe add comment to explain why we need it.

Reply:
There's a brief description in the header file.